pages/tools.md

```diff
@@ -7,31 +7,31 @@ parent: Documentation
 ---
 
 # RSMP Tools
-RSMP Nordic maintains a set of open-source tools for working with RSMP.
+RSMP Nordic maintains a set of open source tools for working with RSMP.
 
-This include both GUI and command-line tools for validation and interacting with RSMP implementations.
+This includes both GUI and command line tools for validation and interacting with RSMP implementations.
 
 ![RSMP Test Stack](/assets/images/test_stack.png)
 
 See all our [GitHub repositories](https://github.com/rsmp-nordic).
 
 ## RSMP Simulator
-The [RSMP Simulator ](https://github.com/rsmp-nordic/rsmp_simulator) is an Windows application written in C#, which can by used for manual testing of RSMP implementations. You can download it along with its documentation from the ![release page](https://github.com/rsmp-nordic/rsmp_simulator/releases).
+The [RSMP Simulator ](https://github.com/rsmp-nordic/rsmp_simulator) is a Windows application written in C# that can be used for manual testing of RSMP implementations. You can download it along with its documentation from the ![release page](https://github.com/rsmp-nordic/rsmp_simulator/releases).
 
 ![RSMP Simulator](/assets/images/simulator_1.0.1.5.png)
 
 ## RSMP Validator
-The [RSMP Validator](https://rsmp-nordic.github.io/rsmp_validator) is a command-line tool based on Ruby and RSpec which can by used for automated validation of RSMP implementations.
+The [RSMP Validator](https://rsmp-nordic.github.io/rsmp_validator) is a command line tool based on Ruby and RSpec that can be used for automated validation of RSMP implementations.
 
 ## RSMP Building Blocks
 We also maintain several Ruby gems (libraries) for working with RSMP.
 
-The RSMP Validator is build on these building blocks, but you can also use them yourself to build new or specialized RSMP tools.
+The RSMP Validator is built on these building blocks, but you can also use them yourself to build new or specialized RSMP tools.
 
 ### RSMP Ruby Gem
 The [RSMP gem](https://github.com/rsmp-nordic/rsmp) is a Ruby library for handling RMSP communication.
 
-It also includes command-line tools that makes it easy to run an RSMP site or supervisor on the command-line and view messages as they are exchanged.
+It also includes command line tools that make it easy to run an RSMP site or supervisor on the command line and view messages as they are exchanged.
```

Comment:
let's use hyphens when command-line is used as an adjective: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/command-line

Reply:
fixed